azuredevops/internal/service/wiki/resource_wiki_test.go

```diff
@@ -0,0 +1,104 @@
+//go:build (all || wiki || resource_wiki) && !exclude_resource_wiki
+// +build all wiki resource_wiki
+// +build !exclude_resource_wiki
+
+package wiki
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v6/wiki"
+	"github.com/microsoft/terraform-provider-azuredevops/azdosdkmocks"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/client"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/converter"
+	"github.com/stretchr/testify/require"
+)
+
+var testWikiProjectID = uuid.New()
+var testWikiID = uuid.New()
+var testWikiType = wiki.WikiType("codeWiki")
+
+func TestWiki_Create_DoesNotSwallowError(t *testing.T) {
```

Comment:
Can split this test into different tests?
1. Test for code wiki
2. Test for project wiki
3. Test for update
4. Test for import